pinc/stages.inc

```diff
@@ -307,7 +307,7 @@ new Stage(
 // -----------------------------------------------------------------------------
 
 new Pool(
-    'PPV',
+    'PPV', // @phpstan-ignore-line PHPStan 1.10 reports a string-to-int conv here, seems like a bug
```

Comment:
is it worth setting up a minimal example and creating an issue against phpstan?

Reply:
Yes, I just did! Also the reproduction showed that the issue is that we reuse the name `Pool` that exists in the PHP standard library and confuses PHPStan:

https://github.com/phpstan/phpstan/issues/11303

Reply:
That's awesome, thanks.